mm2src/coins/eth.rs

```diff
@@ -4504,8 +4551,18 @@ impl EthCoin {
                         },
                     }
                 },
+                EthCoinType::Trx { .. } => {
+                    // TODO use Tron client
+                    warn!(
+                        "Using stub implementation for Tron address_balance for {}, returning {} SUN",
+                        format!("{:?}", tron::TronAddress::from(&address)),
+                        tron::trx_to_sun_u256(0u64).unwrap()
+                    );
+
+                    Ok(U256::zero())
```

Comment:
Panic safe approach:

```suggestion
                    let sun = U256::zero();
                    warn!(
                        "Using stub implementation for Tron address_balance for {}, returning {sun} SUN",
                        format!("{:?}", tron::TronAddress::from(&address)),
                    );

                    Ok(sun)
```